keys/README.md

```diff
@@ -0,0 +1 @@
+This folder contains the identity files.
```

Comment:
So users have to copy/move their SSH keys to this folder for spark-ec2 to be able to use them?